### DISCUSSION TITLE
Minidumps size limit

### DISCUSSION THREAD

**Discussion by WD32** [getsentry/sentry-native#829] [Q&A] [answered]
I am trying to upload native minidumps to Sentry, but some of them have a size larger than 20 MB. According to the documentation, 20 MB is the size limitation for [compressed minidump requests](https://docs.sentry.io/platforms/native/guides/minidumps/#size-limits), and the limit for uncompressed dumps is 100 MB. However, when I compress a minidump to a ZIP file and try to upload it, Sentry returns an error message saying "Invalid minidump". 

I don't understand how to send compressed minidumps and what the documentation means by 'compressed minidump request'. 

What am I doing wrong? Is it possible to upload compressed minidumps?

**Comment by Swatinem** [comment, marked as answer]
100M is pretty much a hard limit that we do not want to raise. The 20M limit is meant as the http `Transfer-Encoding` compression, and is being enforced by relay. I don’t think that limit is flexible either.

**Reply by Swatinem** [reply]
To clarify, it is the `Content-Encoding` encoding header, as documented here: https://develop.sentry.dev/sdk/overview/#request-compression The 20M are the limit for the request body, and the 100M for the decompressed minidump.

I checked back with the team, and neither of these limits will be relaxed anytime soon.
